package.json

```
@@ -27,24 +27,24 @@
     "lib"
   ],
   "devDependencies": {
-    "@types/node": "^22.13.1",
-    "@types/semver": "^7.3.9",
-    "@typescript-eslint/eslint-plugin": "^8.24.0",
-    "@typescript-eslint/parser": "^8.24.0",
-    "eslint": "^8.57.0",
-    "eslint-config-prettier": "^8.5.0",
-    "husky": "^9.1.7",
-    "lint-staged": "^16.0.0",
-    "prettier": "^2.6.2",
-    "typescript": "^5.7.0",
-    "vitest": "^3.1.3"
+    "@types/node": "22.13.1",
+    "@types/semver": "7.3.9",
+    "@typescript-eslint/eslint-plugin": "8.24.0",
+    "@typescript-eslint/parser": "8.24.0",
+    "eslint": "8.57.0",
```

```suggestion
    "eslint": "8.57.1",
```

nit: i think it was this in the lockfile